flow/flow.h

```diff
@@ -1009,6 +1013,9 @@ struct ActorSingleCallback : SingleCallback<ValueType> {
 	virtual void fire(ValueType const& value) {
 		static_cast<ActorType*>(this)->a_callback_fire(this, value);
 	}
+	virtual void fire(ValueType &&value) {
```

Comment:
I don't think this has an effect currently since the callbacks generated by the actor compiler all take const references. Would you be ok with removing this for this PR and possibly including it in a followup that includes the actor compiler changes?

Reply:
This change is necessary for this PR. `a_callback_fire` takes a parameter by value, not by const reference.